lib/rpc/rpc_sidechain.dart

```diff
@@ -220,7 +220,10 @@ class SidechainRPCLive extends ChangeNotifier implements SidechainRPC {
   @override
   Future<List<Transaction>> listTransactions() async {
     // first list
-    final transactionsJSON = await _client?.call('listtransactions', []) as List<dynamic>;
+    final transactionsJSON = await _client?.call('listtransactions', [
+      '',
+      9999,
```

Comment:
could we add a comment explaining that this is the TX count (I assume it is?) and why we're passing such a large number?